cypress/support/commands.js

```diff
@@ -23,19 +23,78 @@
 //
 // -- This will overwrite an existing command --
 // Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
+const imgUrl = 'https://static.productionready.io/images/smiley-cyrus.jpg';
 
-import { addMatchImageSnapshotCommand } from 'cypress-image-snapshot/command';
+Cypress.Commands.add('login', (user) => {
+  cy.request({
+    method: 'POST',
+    url: '/api/users/login',
+    body: {
+      user
+    }
+  }).then((response) => {
+    const user = {
+      bio: response.body.user.bio,
+      effectiveImage: imgUrl,
+      email: response.body.user.email,
+      image: response.body.user.image,
+      token: response.body.user.token,
+      username: response.body.user.username
+    };
+    window.localStorage.setItem('user', JSON.stringify(user));
+    cy.setCookie('auth', response.body.user.token);
+  });
+});
+
+Cypress.Commands.add('createArticle', (user, article) => {
```

Comment:
The `createArticle` command requires both `user` and `article` objects. Ensure that these objects are correctly structured and contain the necessary fields before calling this command.